### ISSUE TITLE
Incorrect indicators

### ISSUE THREAD

**Issue by flashingtt** [chencn2020/MobileIQA#4]
@chencn2020 Thank you for your excellent work!
I tried to use the code you provided to reproduce the evaluation indicators in the paper, but the final result was very unsatisfactory.
Here are the results for MobileVit_IQA on the validation set of UHD-IQA:
`KRCC: 0.2885`	`SRCC: 0.4160`	`PLCC: 0.4196`
Here are my options:
```
--seed 3407 --batch_size 4 -- epochs 100 --dataset uhdiqa --loss MSE --T_max 25 --eta_min 1e-6 --model MobileVit_IQA --mal_num 3 --lr 1e-5 --weight_decay 1e-5
```
Could you please provide some tips, where I overlooked?

**Comment by chencn2020** [comment]
Hi, sorry for late reply.

My guess is that this is caused by the model inputting the image at too low a resolution $$448\times448$$. If you input it at the resolution of the paper ($$1907 \times1231$$), you should get consistent results.

**Comment by flashingtt** [comment]
> Hi, sorry for late reply.
> 
> My guess is that this is caused by the model inputting the image at too low a resolution 448 × 448 . If you input it at the resolution of the paper ( 1907 × 1231 ), you should get consistent results.

Thanks for your reply! I retried with the resolution (1907 × 1231), the situation has improved and the indicators are close to the paper:
`KRCC: 0.5592` `SRCC: 0.7516` `PLCC: 0.7555`
But there is still a certain gap between my results and the results in the paper

**Comment by chencn2020** [comment]
Hi, sorry for late reply.

You can try to adjust the learning rate and epochs. 

And the version of ```timm```  and ```torch``` should be same as the ```requirement.txt``` for better reproduction ~
